ingestion-lambda/src/handler.ts

```diff
@@ -94,16 +100,21 @@ export const main = async (event: SQSEvent): Promise<SQSBatchResponse> => {
 					messageAttributes,
 					body,
 				}): Promise<OperationResult> => {
+					logger.log({
+						message: `Processing message for ${sqsMessageId}`,
+						eventType: INGESTION_PROCESSING_SQS_MESSAGE_EVENT_TYPE,
+						sqsMessageId,
+					});
 					try {
-						const messageId = messageAttributes['Message-Id']?.stringValue;
+						const externalId = messageAttributes['Message-Id']?.stringValue;
```

Comment:
`externalId` helps to differentiate it from the `sqsMessageId`, and is also what we call this value in the database.